src/Main.jsx

```diff
@@ -137,8 +137,10 @@ export const Main = () => {
               }
             />
           </Routes>
+          {/* NB disabled logic to render footer */}
           {/* To avoid CLS, load in the footer after the carousels render */}
-          {carouselLoaded && <Footer />}
+          {/* {carouselLoaded && <Footer />} */}
```

Comment:
Why is this left in at all?

Reply:
Left in because I would eventually like to re-add this functionality
I should've commented